CalibPPS/TimingCalibration/python/ALCARECOPPSTimingCalib_cff.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoPPS.Configuration.recoCTPPS_cff import ctppsDiamondRecHits as _ctppsDiamondRecHits
+from CalibPPS.TimingCalibration.ppsTimingCalibrationPCLWorker_cfi import ppsTimingCalibrationPCLWorker
+
+MEtoEDMConvertPPSTimingCalib = cms.EDProducer('MEtoEDMConverter',
+    Name = cms.untracked.string('MEtoEDMConverter'),
+    Verbosity = cms.untracked.int32(0),
+    Frequency = cms.untracked.int32(50),
+    MEPathToSave = cms.untracked.string('AlCaReco/PPSTimingCalibrationPCL'),
+    deleteAfterCopy = cms.untracked.bool(True),
+)
+
+# calibrated rechits/tracks
+ctppsDiamondUncalibRecHits = _ctppsDiamondRecHits.clone(
+    applyCalibration = False
+)
+recoDiamondUncalibLocalReconstructionTask = cms.Task(
+    ctppsDiamondUncalibRecHits,
+)
+# this sequence will be updated to include tracking based on the last
+# calibration values to extract per-channel timing precision estimation
+recoDiamond = cms.Sequence(recoDiamondUncalibLocalReconstructionTask)
+
+seqALCARECOPPSTimingCalib = cms.Sequence(
+    recoDiamond *
+    ppsTimingCalibrationPCLWorker *
+    MEtoEDMConvertPPSTimingCalib
+)
```

Comment:
please use Task

Reply:
For this part, I shamelessly stole tracker and various muon chambers PCL's chunks. Would this `Sequence` → `Task` conversion be 1-to-1 compatible?

Reply:
yes, it should be compatible. Did you get error? (see https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#Task_Objects)

Reply:
They are not. EDAnalyzers have to be put in a Sequence, as do EDFilters whose event filtering decision is not ignored (EDFilters can be placed in Task but the filtering is ignored). An EDProducer placed in Task is run only if some other module that consumes its product is run.

If all these modules are EDProducers, a Task is better. If there are EDFilters that are expected to control the running of the EDProducers (in this Sequence or in any other Sequence in the same Path), Sequence must be used, and the modules should not be placed in any Task in the same job (technically they can but it doesn't have any effect and therefore can be confusing).

Reply:
Many thanks for the clarification, @makortel !
Do I understand correctly that the `seqALCARECOPPSCalTrackBasedSel` (containing the `ALCARECOPPSCalTrackBasedSelHLT` EDFilter) should be reverted to a `Sequence` while this `seqALCARECOPPSTimingCalib` can indeed be converted to a `Task` as it only contains EDAnalyzer/EDProducers?

Reply:
Right, the seqALCARECOPPSTimingCalib could be changed to a Task (the `PPSTimingCalibrationPCLWorker` is an `DQMGlobalEDAnalyzer` that is really and EDProducer).

Reply:
Changed in https://github.com/cms-sw/cmssw/pull/33215/commits/f9c1fcdef51e305ba980e8ce13e29ed61c2a5167. Thanks for the review!